gzweb_install/tutorial.md

```diff
@@ -6,38 +6,53 @@ on a web browser.
 
 # Dependencies
 
-The main dependencies for GzWeb are the Gazebo development libraries, version 7 or
-greater, and NodeJS version 4 up to version 8.
+The main dependencies for GzWeb are the Gazebo development libraries, version 9 or
+greater, and NodeJS version 6 or greater.
 
 Take a look at
 [these tutorials](http://gazebosim.org/install) to choose the Gazebo
 installation that best fits your case. The simplest approach would be
-to install Gazebo 7 as follows:
+to install Gazebo 9 as follows:
 
 ~~~
-sudo apt install gazebo7 libgazebo7-dev
+sudo apt install gazebo9 libgazebo9-dev
 ~~~
 
 Run the following to install the rest of dependencies, including NodeJS:
 
 ~~~
-sudo apt install libjansson-dev nodejs npm nodejs-legacy libboost-dev imagemagick libtinyxml-dev mercurial cmake build-essential
+sudo apt install libjansson-dev libboost-dev imagemagick libtinyxml-dev mercurial cmake build-essential
 ~~~
 
-> Ubuntu Trusty comes with NodeJS 0.10. You can follow
-  [these](https://github.com/nodesource/distributions) instructions to upgrade
-  the Node version.
+Install nodejs
+
+~~~
+sudo apt install nodejs npm
+~~~
```

Comment:
This gave me issues on Bionic, and I didn't try it on Focal. There were conflicts with the `libssl` version needed by Gazebo and by Node. Are you sure it may work?

Reply:
I ran into the same issue but I wasn't sure if it's just me so I added the `nvm` option below. Looks like it's a common issue. I'll remove the `apt install` option

Reply:
updated. bcf9177